src/main/java/ConsoleWriter.java

```diff
@@ -7,11 +7,11 @@ public ConsoleWriter(PrintStream output) {
         this.output = output;
     }
 
-    public ConsoleWriter(){
+    public ConsoleWriter() {
         this(System.out);
     }
 
-    public void println(String output){
+    public void println(String output) {
         this.output.println(output);
```

Comment:
You don't need the `this` prefix unless you also have an argument or a variable declared with the same name. Generally I prefer to avoid using `this` unless it's required.

Reply:
Thanks, Matt. I was experiencing issues when not using `this`, due to the name of the parameter. I'll change the parameter name